.github/dependabot.yml

```diff
@@ -1,6 +1,26 @@
 version: 2
+
 updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "weekly"
+    cooldown:
+      default-days: 7
+  - package-ecosystem: "gradle"
+    directory: "/"
+    registries:
+      - google
+      - gradle-plugin
+    schedule:
+      interval: "weekly"
+    cooldown:
+      default-days: 7
+
+registries:
+  google:
+    type: maven-repository
+    url: maven.google.com
```

Comment:
Google's Maven Repository has to be specified manually, since Dependabot only checks Apache's Maven Repository by default.